keyboards/handwired/oled25/rules.mk

```diff
@@ -0,0 +1,2 @@
+OLED_ENABLE     = yes
```

Comment:
If OLEDs are enabled at the keyboard level, functionality for them should exist at the keyboard level as well.
eg. in a `oled25.c` file using the `oled_task_kb()` function.

Reply:
If OLEDs are meant to be installed, functionality for them should be present at the keyboard level. There are lots of keyboards that do this — you can run `grep -r "oled_task_kb" keyboards` from within your `qmk_firmware` directory to list them.